src/v2/Apps/Partner/Components/PartnerContacts/PartnerContactAddress.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react"
+import { Text } from "@artsy/palette"
+import { getContactAddressLines } from "./partnerContactUtils"
+import { PartnerContactAddress_location } from "v2/__generated__/PartnerContactAddress_location.graphql"
+import { createFragmentContainer, graphql } from "react-relay"
+
+export interface PartnerContactAddressProps {
+  location: PartnerContactAddress_location
+}
+
+export const PartnerContactAddress: React.FC<PartnerContactAddressProps> = ({
+  location,
+}) => {
+  const addressLines = getContactAddressLines(location)
+
+  return (
+    <>
+      {addressLines.map((line, index) => (
+        <Text key={index}>{line}</Text>
+      ))}
+      {location.phone && <Text mt={1}>{`Tel: ${location.phone?.trim()}`}</Text>}
+    </>
+  )
+}
+
+graphql`
+  fragment PartnerContactAddress_Fragment on Location {
+    city
+    phone
+    state
+    country
+    address
+    address2
+    postalCode
+  }
+`
+
+export const PartnerContactAddressFragmentContainer = createFragmentContainer(
+  PartnerContactAddress,
+  {
+    location: graphql`
+      fragment PartnerContactAddress_location on Location {
+        ...PartnerContactAddress_Fragment @relay(mask: false)
```

Comment:
What's with the indirection here?

Reply:
I'm not sure that I understand you correctly. Do you mean why I added the fragment(PartnerContactAddress_Fragment)? if so, then I created PartnerContactAddress_Fragment to use it in two places: here and in PartnerContactMap_location fragment.

Reply:
But why is that? It should just be the case that the fields are redeclared where they are used rather than have them be shared.

Reply:
Well, the redeclared field seems ok to me, but the main thing of my approach was DRY conception. If it is okay I will fix it just with redeclared fields

Reply:
Yeah, one of the core principles with Relay is the co-location of the data requirements with the view. You're not really repeating yourself — you're clearly stating the data that a given component or function requires inline with where it lives.

Reply:
Updated, please take a look.

Reply:
Perfect, thank you!